src/databricksbundle/_config/config.yaml

```diff
@@ -1,7 +1,9 @@
 parameters:
+  daipehelpargs:
+    daipecore: daipe-core
```

Comment:
does this need to be here?
it's not like anyone is going to override that
just put it as a constant in the `DaipeHelp` file